### PR TITLE
switch to using pandas-dev/pandas-stubs

### PR DESCRIPTION
**Pull request by Dr-Irv**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Switching from https://github.com/VirtusLab/pandas-stubs to https://github.com/pandas-dev/pandas-stubs

### CODE REVIEW

**Comment by zkrolikowski-vl** [comment]
```
RuntimeError: SHA256 mismatch: '89c022dad41d28e26a1290eb1585db1042ccb09e6951220bb5c9146e2f795147' != 'a9096caa76c3705be7ce5a14e6881bb8e8c2cd097da4b2d9376260f584f36be5'
```

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,40 +7,37 @@ package:
...
-    - typing_extensions >=3.7.4.3  # [py<=38]
+    - poetry
+    - typing_extensions >=4.2  # [py>=38]
   run:
     - python
-    - typing_extensions >=3.7.4.3  # [py<=38]
+    - poetry
+    - typing_extensions >=4.2  # [py>=38]
 
-test:
```
Comment (zkrolikowski-vl) [inline recipe/meta.yaml:29]: Why were the test commands removed?

**Comment by Dr-Irv** [comment]
> ```
> RuntimeError: SHA256 mismatch: '89c022dad41d28e26a1290eb1585db1042ccb09e6951220bb5c9146e2f795147' != 'a9096caa76c3705be7ce5a14e6881bb8e8c2cd097da4b2d9376260f584f36be5'
> ```

Ooops.  I forgot to change the version number!

File: recipe/meta.yaml (hunk 1 above)
Comment (Dr-Irv) [reply recipe/meta.yaml:29]: The tests you had were not right for the way we are installing `pandas-stubs`, which is as a separate package.  Trying to figure out what kind of test to use.

**Pull request merged**